test/gurka/test_gtfs.cc

```diff
@@ -122,23 +122,23 @@ TEST(GtfsExample, WriteGtfs) {
   feed.write_agencies(path_directory);
 
   // write stops.txt:
-  // 1st has all stop objects, egress/station/platform
-  // 2nd has only platform
+  // 1st has all stop objects, egress/station/platform, but egress has no parent
+  // 2nd has only platform, station, but the platform has not parent
   // 3rd has only station
```

Comment:
nit: maybe im reading the diff wrong but it seems like the comments here arent exactly right and the 2nd stop is deliberately incorrect? what i see is:

* 1st has all stop types, egress/station/platform (the egress and platform do point to the station as the correct parent parent)
* 2nd has station/platform only (the platform does not point to the parent station, so the station is orphaned, why do we test this? shouldnt we point the platform to the parent station?)
* 3rd has only a platform

Reply:
> so the station is orphaned

Right, but the comment is actually saying that.

> why do we test this?

I realized a bit too late that this is bs actually. What would you like to see here? I had only a platform before without a station for the second one, but you wanted to see more when we reviewed. Nothing really comes to my mind that would make more sense tbh. An egress without a parent station could be, but that's pretty much the same as an orphaned station.